src/promptflow-tools/promptflow/tools/embedding.py

```diff
@@ -10,6 +10,14 @@
 from promptflow.connections import AzureOpenAIConnection, OpenAIConnection
 
 
+def is_serverless_connection(connection):
```

Comment:
Use this compatible process? Then you can keep using `isinstance`.
try:
    from promptflow.connections import ServerlessConnection
except ImportError:
    # If unable to import ServerlessConnection, define a placeholder class to allow isinstance checks to pass.
    # ServerlessConnection was introduced in pf version 1.6.0.
    class ServerlessConnection:
        pass